### PR TITLE
run rustfmt on syntax::parse::lexer

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/libsyntax/parse/lexer/comments.rs [hunk 1]
```diff
@@ -64,18 +62,18 @@ pub fn strip_doc_comment_decoration(comment: &str) -> String {
...
         while i < j && lines[i].trim().is_empty() {
             i += 1;
         }
         // like the first, a last line of all stars should be omitted
-        if j > i && lines[j - 1]
-                         .chars()
-                         .skip(1)
-                         .all(|c| c == '*') {
+        if j > i &&
+           lines[j - 1]
```
Comment (nagisa) [inline src/libsyntax/parse/lexer/comments.rs]: Downgrade here.

File: src/libsyntax/parse/lexer/comments.rs [hunk 2]
```diff
@@ -165,19 +162,25 @@ fn consume_whitespace_counting_blank_lines(rdr: &mut StringReader,
...
+                        code_to_the_left: bool,
                         comments: &mut Vec<Comment>) {
     debug!(">>> shebang comment");
     let p = rdr.last_pos;
     debug!("<<< shebang comment");
     comments.push(Comment {
-        style: if code_to_the_left { Trailing } else { Isolated },
-        lines: vec!(rdr.read_one_line_comment()),
-        pos: p
+        style: if code_to_the_left {
```
Comment (nagisa) [inline src/libsyntax/parse/lexer/comments.rs]: IMO it was better the previous way here.

File: src/libsyntax/parse/lexer/comments.rs [hunk 3]
```diff
@@ -195,9 +198,13 @@ fn read_line_comments(rdr: &mut StringReader, code_to_the_left: bool,
     debug!("<<< line comments");
     if !lines.is_empty() {
         comments.push(Comment {
-            style: if code_to_the_left { Trailing } else { Isolated },
+            style: if code_to_the_left {
```
Comment (nagisa) [inline src/libsyntax/parse/lexer/comments.rs]: ditto.

**Comment by nagisa** [comment]
I also have a few nits for parse/mod.rs changes, but Github won’t show the file for me to comment on.

File: src/libsyntax/parse/lexer/comments.rs (hunk 2 above)
Comment (sanxiyn) [inline src/libsyntax/parse/lexer/comments.rs]: I agree. I opine that this should be fixed in rustfmt and formatting left alone here.

**Comment by brson** [comment]
Waiting for rebase or rerun.

**Comment by tshepang** [comment]
Too messy to do a rebase. Will do a re-run.

**Comment by tshepang** [comment]
The too-many conflicts led me to create a new PR: https://github.com/rust-lang/rust/pull/30684

**Pull request closed without merging**